src/luascript.cpp

```diff
@@ -12260,7 +12260,7 @@ int LuaScriptInterface::luaMonsterTypeIsSummonable(lua_State* L)
 	MonsterType* monsterType = getUserdata<MonsterType>(L, 1);
 	if (monsterType) {
 		if (lua_gettop(L) == 1) {
-			pushBoolean(L, monsterType->info.isAttackable);
+			pushBoolean(L, monsterType->info.isSummonable);
 		} else {
 			monsterType->info.isAttackable = getBoolean(L, 2);
```

Comment:
replace:

 monsterType->info.isAttackable = getBoolean(L, 2);

for

 monsterType->info.isSummonable= getBoolean(L, 2);